worlds/tunic/er_scripts.py

```diff
@@ -157,7 +167,10 @@ def pair_portals(world: "TunicWorld") -> Dict[Portal, Portal]:
     start_region = "Overworld"
     connected_regions.update(add_dependent_regions(start_region, logic_rules))
 
-    plando_connections = world.multiworld.plando_connections[world.player]
+    if world.options.entrance_rando.value in EntranceRando.options:
+        plando_connections = world.seed_groups[world.options.entrance_rando.value]["plando"]
+    else:
+        plando_connections = world.multiworld.plando_connections[world.player]
```

Comment:
```suggestion
    if world.options.entrance_rando.value in EntranceRando.options:
        plando_connections = world.multiworld.plando_connections[world.player]
    else:
        plando_connections = world.seed_groups[world.options.entrance_rando.value]["plando"]
```

These conditions appear to be inverted, flipping them around fixes the issue in the test case I sent you.

Reply:
Great catch, thank you!